<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_image">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_model">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载模型</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="image_path">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resize_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>尺寸缩放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image_gray">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="median_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>中值滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="features_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>特征提取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="model_predict">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>模型预测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handle_image">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>一键处理</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>预测结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>58</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-----</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
